Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM alpine:3.5
+FROM python:3.6-alpine
```

Comment:
`python:3.6-alphine` image is vulnerable to attacks. Can you consider updating to `python:3.9.12-alpine3.15` and testing it?